docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -119,16 +119,20 @@ Not compatible with Sauce Connect versions below 4.7.0. Download the latest vers
 ### `--direct-domains`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__: Sets domain(s) that you want to relay directly through the internet instead of through the Sauce Connect Proxy tunnel. When adding multiple domains, [format as a comma-separated list](#formatting-domains-in-the-command-line).<br/>
+__Description__: Sets domain(s) that are requested through the public internet instead of the Sauce Connect Proxy tunnel. This is the inverse of [--tunnel-domains](#--tunnel-domains). When adding multiple domains, [format as a comma-separated list](#formatting-domains).<br/>
```

Comment:
```suggestion
__Description__: Sets domain(s) that are requested through the public internet instead of the Sauce Connect Proxy tunnel. This is the inverse of [`--tunnel-domains`](#--tunnel-domains). When adding multiple domains, [format as a comma-separated list](#formatting-domains).<br/>
```